doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -1420,14 +1438,19 @@ The connection method for internal network servers in the **service provider rol
 
 4.  Click on a service code in the table.
     
-5.  In the view that opens, change the protocol in the service URL to HTTP, HTTPS or HTTPS NO AUTH.
-    If the HTTPS protocol was selected, select the **Verify TLS certificate** checkbox if needed (see section [6.6](#66-changing-the-parameters-of-a-service)). According to the service parameters, the connection with the internal network server is created using one the following protocols:
+5.  In the view that opens, change the protocol (a.k.a. scheme) part in the Service URL to either **http://** or **https://**.
```

Comment:
I would consider updating this part a little bit:

> 5. In the view that opens, change the protocol (a.k.a. scheme) part in the Service URL to either `http://` or `https://`.
> - HTTP – the service/adapter URL begins with `http://`.
> - HTTPS – the service/adapter URL begins with `https://`.
>    - If **Verify TLS certificate** checkbox is left unchecked it means that service provider information system's TLS certificate is not verified and is trusted by default.
 >   - If **Verify TLS certificate** checkbox is checked it means that service provider information system's TLS certificate is verified. In order to make the information system's TLS certificate trusted, it must be added into security server's **Information System TLS certificate** list (see section [9.3](https://github.com/nordic-institute/X-Road/blob/00c191c0670c231d295beb967826bcaa52bdae38/doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md#93-managing-information-system-tls-certificates)).
>    - When the service provider information system needs to verify the Security Server's internal TLS certificate, the certificate must be first exported and then imported into the service provider information system's truststore (see section [9.3](https://github.com/nordic-institute/X-Road/blob/00c191c0670c231d295beb967826bcaa52bdae38/doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md#93-managing-information-system-tls-certificates)).